tests/infrastructure/golden_images/test_common_templates_data_volumes.py

```diff
@@ -167,8 +167,8 @@ def test_vm_with_existing_dv(data_volume_scope_function, vm_from_template_with_e
 )
 @pytest.mark.polarion("CNV-5529")
```

Comment:
Should you mark these tests with storage class names?

Reply:
you mean consider having add storage markers nfs and HOSTPATH_CSI_BASIC specifically?

Reply:
yes. If those are required by this test, then it should not be collected unless the lane has nfs (Which we don't test currently) and HPP CSI.

Reply:
@dbasunag okay, Jenia too asked something similar, I'll wait for her response.